styles.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "inter", sans-serif;
+  color: white;
+}
+
+div {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+nav {
+  height: 10vh;
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  width: auto;
+  margin: 0;
+}
+
+.logo-text {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  padding-left: 24px;
+}
+
+.menu {
+  width: 40px;
+  height: 40px;
+  display: block;
+  align-self: center;
+  align-content: center;
+  padding: 10px 10px;
+  float: right;
+  margin-right: 20px;
+}
+
+#check {
+  display: none;
+}
+
+.main-line {
```

Comment:
:red_circle: You have well implementing some of the requested changes for this section Kindly notice that the header element still has a bigger font-weight than the expect one. The Figma Design use a font-weight of 400. 

|Your Design | Figma Design|
|:----------------:|:-----------------:|
|![desi](https://user-images.githubusercontent.com/26736582/212360811-6e00cf80-ba2f-4e75-a04a-adb3cea89317.png)|<img src="https://user-images.githubusercontent.com/99246359/212210054-ba430b24-a4b8-44c5-b206-6abf456352ba.png"/>|

:bulb:  hint: You can create a CSS selector with this rule and apply it to your element as shown here :arrow_down: 
```
.fw-400 {
   font-weight: 400;
}
```
Then apply it to your h1 as follow

` <h1 class="orange fw-400">Hey There.<br>I'm Matee</h1>`